src/agent/debuglet.js

```diff
@@ -736,3 +735,11 @@ Debuglet._delimit = function(source, delim) {
   }
   return dest;
 };
+
+Debuglet._createUniquifier = function (desc, version, uid, sourceContext,
+  labels) {
+  var uniquifier = desc + version + uid + JSON.stringify(sourceContext) +
+    JSON.stringify(labels);
+  uniquifier = crypto.createHash('sha1').update(uniquifier).digest('hex');
```

Comment:
Nit: could we return `crypto.createHash....` directly to avoid reassigning just to return?